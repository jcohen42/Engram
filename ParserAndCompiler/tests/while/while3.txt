Begin program:
	Set x equal to 6.
	While x is equal to 6:
		Set x equal to 7.
		Output "In while".
	End while.
	Output "Out of while".
End program.