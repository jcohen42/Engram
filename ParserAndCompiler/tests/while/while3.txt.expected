In while
Out of while